src/basho_bench_stats_writer_csv.erl

```diff
@@ -186,16 +193,15 @@ dump_run_statistics(RunStatsType) ->
 stringify_stats(_RunStatsType=json, RunMetrics) ->
     [ jiffy:encode( {[{recordedMetrics, {RunMetrics}}]}, [pretty] ) ];
 stringify_stats(_RunStatsType=csv, RunMetrics) ->
-    %% Generate list of every known header
-    Headers = lists:foldl(fun(K, Acc) ->
-        {MetricsObj} = proplists:get_value(K, RunMetrics),
-        proplists:get_keys(MetricsObj) ++ Acc
-    end, [], proplists:get_keys(RunMetrics)),
+    % Ordered output of fields
+    OrderedFields = [mean, geometric_mean, harmonic_mean,
+        variance, standard_deviation, skewness, kurtosis,
+        median, p75, p90, p95, p99, p999,
+        min, max, n, basho_errors],
```

Comment:
Let's move `n` to the front of this list so that the rows start with `op, n, ...`. This will more closely lineup with the existing output format that puts `n` ahead of the rest of the stats.